hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/EncryptedFileSystem.java

```diff
@@ -97,10 +97,10 @@ public FSDataOutputStream create(Path path, FsPermission permission,
     @Override
     public FSDataOutputStream create(Path path, FsPermission permission, EnumSet<CreateFlag> flags, int bufferSize,
             short replication, long blockSize, Progressable progress, ChecksumOpt checksumOpt) throws IOException {
-        FSDataOutputStream encryptedStream =
+        FSDataOutputStream outputStream =
```

Comment:
Same thing for the other `create` method above this one

Reply:
done